src/mm-dialog/mm-dialog.scss

```diff
@@ -0,0 +1,16 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+@import "_bourbon";
+@import "_color";
+@import "_mixins";
+
+@include fit();
+
+.content-top {
```

Comment:
I'd give .content-top a padding of about 30px, so it feels less claustrophobic. This also follows the current T1 implementation.